client/index_template.html

```diff
@@ -38,9 +38,11 @@
   <body>
     <script type="text/javascript">
       window.CELLXGENE = {};
+      backend_base_url = "{{BACKEND_BASE_URL}}";
```

Comment:
Prefer JS convention of camelCase `backendBaseURL`.

Is this supposed to be a string literal?  `backend_base_url` will _always_ `===` the string`{{BACKEND_BASE_URL}}`

This means that the following ternary will never assign `location.origin` to `prefix`

Reply:
This is a template.  The BACKEND_BASE_URL gets replaced with the what the server passed when it rendered the template.  Other examples of this are SCRIPTS and INLINE_SCRIPTS, which are handled differently because they are lists.

Reply:
Ah I thought we were using a different templating engine and not familiar with lodash'. I thought that it wouldn't replace any text inside the literal string.  LGTM!

Reply:
I changed to camelCase